demo/src/main/java/com/eis/geoCalendar/demo/MoveMapCommand.java

```diff
@@ -0,0 +1,25 @@
+package com.eis.geoCalendar.demo;
+
+import android.location.Location;
+
+import com.eis.geoCalendar.demo.Localization.Command;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+
+/**
+ * Used to move the Map to the defined Location, with the max possible zoom
+ */
+public class MoveMapCommand implements Command<Location> {
+    private GoogleMap googleMap;
+
+    public MoveMapCommand(GoogleMap map){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4824932)